src/App.vue

```diff
@@ -233,11 +234,23 @@ export default {
 					this.refreshCurrentConversation()
 				} else {
 					console.info('Conversation received, but the current conversation is not in the list. Redirecting to /apps/spreed')
-					this.$router.push('/apps/spreed/not-found')
+					EventBus.$emit('conversationNotFound')
 				}
 			}
 		})
 
+		EventBus.$on('conversationNotFound', () => {
+			// this will skip showing a warning during a call and
+			// also skip auto-leaving the current conversation, which does not exist anyway
+			this.isLeavingAfterSessionConflict = true
+			SessionStorage.removeItem('joined_conversation')
+			this.$nextTick(() => {
+				// Need to delay until next tick, otherwise the PreventUnload is still being triggered
+				// Putting the window in front with the warning and irritating the user
+				window.location = generateUrl('/apps/spreed/not-found')
```

Comment:
Why this over `this.$router.push('/apps/spreed/not-found')` ?

Reply:
Also this doesn't seem to work. For me the "Leave call" popup is still shown (without HPB)

Reply:
ah, I copied this from mixins/duplicateSessionHandler which also doesn't use router push.
I'll check if it works with that.

Reply:
> Also this doesn't seem to work. For me the "Leave call" popup is still shown (without HPB)

It works if the deletion is done outside the current browser window as per original reproduction steps.

But indeed if you delete it within the same window you get the warnings. I'll try to add special handling for that scenario.

Reply:
I've tried using router.push() but it's triggering the "you are leaving call" dialog.
When adding a flag to inhibit it, it will display the not found page but with a lot of JS errors...

I don't think it's worth trying to protect against this at this stage (but worth as part of improving the not found page separately).

I'll leave window.location here then. I'm sure this must have been the reason why it was used also for the duplicate session case.

Reply:
Well signaling.js is not Vue, so it can't use router.push()
It smells so trappy to change this. There must be a reason we did it like this and not do a page reload all the time.

Reply:
> It smells so trappy to change this.

I'd appreciate a more diplomatic tone...

I'll have another look and compare with the original behavior and see why the errors I saw didn't happen there.